data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/dao/ScalarDBManager.java

```diff
@@ -0,0 +1,67 @@
+package com.scalar.db.dataloader.core.dataimport.dao;
+
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.api.DistributedTransactionAdmin;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.service.StorageFactory;
+import com.scalar.db.service.TransactionFactory;
+import java.io.IOException;
+
+/**
+ * A manager to retrieve the various ScalarDB managers based on the running mode
+ *
+ * @author Yves Peckstadt
+ */
+public class ScalarDBManager {
+
+  /* Distributed storage for ScalarDB connection that is running in storage mode. */
+  private final DistributedStorage storage;
```

Comment:
These should have `@Nullable` annotations.

BTW, maybe this can be divided into `ScalarDbStorageManger` and `ScalarDbTransactionManger` ?